### PR TITLE
changed to gridlayout and fix widgets not showing

### PR DESCRIPTION
**Pull request by KelvinLWang**

### CODE REVIEW

**Review by ryanwmao — APPROVED** [review]
looks good, maybe comments deleted

File: GUI/utils/header.py [hunk 1]
```diff
@@ -35,8 +35,14 @@ def __init__(self, w=1000, h=500, *args, **kwargs):
         grid1.addWidget(emergency_button, 0, 4,
                         alignment=Qt.AlignCenter)
 
-        self.pBar = Progress(self.width, self.height)
-        grid1.addWidget(self.pBar, 0, 1, alignment=Qt.AlignCenter)
+        # self.pBar = Progress(self.width, self.height)
+        # grid1.addWidget(self.pBar, 0, 1, alignment=Qt.AlignCenter)
```
Comment (roninsharma25) [inline GUI/utils/header.py:39]: These can be deleted

File: GUI/utils/header.py [hunk 2]
```diff
@@ -68,15 +74,15 @@ def __init__(self, w=1000, h=500, *args, **kwargs):
...
+        # splitter4 = QSplitter(Qt.Horizontal)
+        # self.pBarContainer = progressBar.ProgressBar()
+        # splitter4.addWidget(self.pBarContainer.label)
 
-        # PROGRESS BAR
-        splitter4.addWidget(self.pBarContainer.pBar)
-        splitter4.setSizes([int(self.height / 30), int(self.height / 30)])
+        # # PROGRESS BAR
+        # splitter4.addWidget(self.pBarContainer.pBar)
+        # splitter4.setSizes([int(self.height / 30), int(self.height / 30)])
```
Comment (roninsharma25) [inline GUI/utils/header.py:83]: These can also be deleted

File: GUI/widgets/plot_buttons.py [hunk 3]
```diff
@@ -15,56 +16,64 @@ def __init__(self, plot):
...
     def initUI(self):
 
-        hbox = QHBoxLayout()
+        grid = QGridLayout()
 
         # Reset plot button
         self.reset_plot_button = QPushButton(self)
         self.reset_plot_button.setText("Reset Plot")
-        self.reset_plot_button.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
+        self.reset_plot_button.setStyleSheet(
```
Comment (roninsharma25) [inline GUI/widgets/plot_buttons.py:24]: Can you notify the people working on the CSS ticket about these changes? Not sure if they already branched off.

**Review by roninsharma25 — CHANGES_REQUESTED** [review]

**Review by chp36 — APPROVED** [review]

**Review by aislinnennis — APPROVED** [review]

**Review by roninsharma25 — APPROVED** [review]

**Pull request merged**